### DISCUSSION TITLE
Can anyone illustrate how to use this program?

### DISCUSSION THREAD

**Discussion by Sugarman191** [codeacme17/examor#48] [Q&A] [answered]
I am new to coding, and I have already downloaded this program on my computer. However, I don't know how to run it. I really want to try it, as I have been searching for such program for so long. 

I‘d really appreciate some help.

**Comment by codeacme17** [comment, marked as answer]
Hi, You need install docker

**Reply by Sugarman191** [reply]
I have done this with ChatGPT's help😅
